### PR TITLE
break circular imports

### PR DESCRIPTION
**Pull request by freben**

### CODE REVIEW

File: plugins/config-schema/src/components/SchemaView/SchemaView.tsx [hunk 1]
```diff
@@ -15,12 +15,22 @@
...
+);
+const MatchView = React.lazy(() =>
+  import('./MatchView').then(m => ({ default: m.MatchView })),
+);
+const ObjectView = React.lazy(() =>
+  import('./ObjectView').then(m => ({ default: m.ObjectView })),
+);
+const ScalarView = React.lazy(() =>
+  import('./ScalarView').then(m => ({ default: m.ScalarView })),
+);
```
Comment (Rugvip) [inline plugins/config-schema/src/components/SchemaView/SchemaView.tsx:32]: so these are a bit strange, how did you arrive at this? 😁

Comment (freben) [reply plugins/config-schema/src/components/SchemaView/SchemaView.tsx:32]: I'm not sure! Haha :)

Like, these are truly circularly importing each other and the build warns about that. I just wanted to fix it somehow. Is this approach a problem?

Comment (vinzscam) [reply plugins/config-schema/src/components/SchemaView/SchemaView.tsx:32]: perhaps try to merge the recursive components in a single file?

Comment (freben) [reply plugins/config-schema/src/components/SchemaView/SchemaView.tsx:32]: Sure. But if not merging all that into one file, I'd still be curious if the lazy thing is unexpected or doesn't work well 🤔

**Review by vinzscam — APPROVED** [review]

**Review by Rugvip — APPROVED** [review]

**Review by Rugvip — APPROVED** [review]

**Pull request merged**